src/Properties.def.js

```diff
@@ -420,7 +455,8 @@
 
             elevatedMethod = {};
             elevatedMethod[methodName] = baseMethod.bind(this);
-            $oop.Base.addMethods.call(this, elevatedMethod);
+
+            self.addProperties.call($oop.Base.getTarget.call(this), elevatedMethod, false, true, false);
```

Comment:
There is a question over elevation of methods. Should a private method ever be allowed to be elevated? Currently there are cases both in our app but also in Giant where this happens. 

This change is required to prevent major backwards incompatibilities - but I think it should be reviewed.